### PR TITLE
gazebo_ros_wheel_slip: publish member variable

### PR DESCRIPTION
**Pull request by scpeters**
I have observed some problems during teardown if gzserver receives a SIGINT while publishing the wheel slip. To try to fix it, this stores the message to be published as a class member rather than a stack variable.

Signed-off-by: Steve Peters <scpeters@openrobotics.org>

### CODE REVIEW

File: gazebo_plugins/src/gazebo_ros_wheel_slip.cpp [hunk 1]
```diff
@@ -385,8 +389,8 @@ void GazeboRosWheelSlip::Load(gazebo::physics::ModelPtr _model, sdf::ElementPtr
       IGN_PROFILE_BEGIN("fill ROS message");
     #endif
 
-      // Populate message
-      auto slip_msg = gazebo_msgs::msg::WheelSlip();
+      // Reset message
+      slip_msg = gazebo_msgs::msg::WheelSlip();
```
Comment (mallanmba) [inline gazebo_plugins/src/gazebo_ros_wheel_slip.cpp]:
```suggestion
      impl_->slip_msg = gazebo_msgs::msg::WheelSlip();
```

Comment (mallanmba) [reply gazebo_plugins/src/gazebo_ros_wheel_slip.cpp]: it won't let me comment on the lines below, but all occurrences of `slip_msg` need to be `impl_->slip_msg`

**Review by adityapande-1995 — APPROVED** [review]
Looks good, wheelslip tests passed CI.

**Comment by scpeters** [comment]
@ros-pull-request-builder test this please

**Pull request merged**